inter-server-sync.spec

```diff
@@ -40,9 +40,14 @@ Source0:        %{name}-%{version}.tar.gz
 Source1:        vendor.tar.gz
 BuildRequires:  golang-packaging
 %if 0%{?rhel}
-BuildRequires:  golang >= 1.16
+BuildRequires:  golang >= 1.17
 %else
-BuildRequires:  golang(API) = 1.16
+%if 0%{?is_opensuse}
+BuildRequires:  golang(API) = 1.18
+%else
+BuildRequires:  go1.18-openssl
+BuildRequires:  golang(API) = 1.18-openssl
```

Comment:
This line seems to be wrong to me. The API version which is provided by both `go1.18` and `go1.18-openssl` equals `1.18`.

Reply:
Agree. I will remove this line